be/src/runtime/exec_env.cpp

```diff
@@ -187,7 +187,7 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
                             .set_max_queue_size(1000)
                             .set_idle_timeout(MonoDelta::FromMilliseconds(2000))
                             .build(&driver_executor_thread_pool));
-    _driver_executor = new pipeline::GlobalDriverExecutor(std::move(driver_executor_thread_pool), false);
+    _driver_executor = new pipeline::GlobalDriverExecutor("pip_exe", std::move(driver_executor_thread_pool), false);
```

Comment:
`exec`?

Reply:
I want the pipeline-related metric to have special keywords so that we can find all the pipeline-related metric from the metrics